torchbenchmark/models/speech_transformer/config.py

```diff
@@ -34,7 +34,7 @@ class SpeechTransformerTrainConfig:
     label_smoothing = 0.1
     # minibatch
     shuffle = 1
-    batch_size = 16
+    batch_size = 32
```

Comment:
Who is using this value? It looks to be replaced by train_bs instead.

Reply:
No, this value is not used. I will remove this. Thanks for pointing out!